src/en/space-station-14/departments/medical/proposals/brigmedic.md

```diff
@@ -0,0 +1,55 @@
+
+*Apr 11, 2023 - Apr 16, 2023:* Brigmedic was mapped onto Bagel, Origin, and Box.
+
+*[Apr 15, 2023](https://github.com/space-wizards/space-station-14/pull/15426):* Brigmedic recieves a nerf, removing their hypospray (hyposprays are currently exclusive to the CMO, Nukie Agent, and CentComm roles).
+
+*[Apr 26, 2023](https://github.com/space-wizards/space-station-14/pull/15496):* An update for brigmedic is merged in, adding in further entities made exclusively for the brigmedic.
+
+*[May 4, 2023](https://github.com/space-wizards/space-station-14/pull/16069):* Brigmedic is removed from the game, citing complaints from Discord as the reason.
+
+*[May 4, 2023](https://github.com/space-wizards/space-station-14/pull/16082):* Brigmedic removal is partially reverted to keep the uniform around.
+
+*[May 6, 2023](https://github.com/space-wizards/space-station-14/pull/16196):* A PR to revert the brigmedic removal is proposed, citing the majority reactions against the removal. The request is rejected, giving the following reason:
+
+> […] we're not readding the free hardsuit role. The game already has one of those (Paramedic) and it already causes issues. This is the same role except with security access to boot with none of the responsibilities of a medic, and it's primary use seemed to be security with a hardsuit, not an actual medic.
+
+> In other words: Gimmie a well reasoned, thought out design doc for this role, what it should do, what it should not do and how it makes sure those things don't happen, and how it should influence the round, and I'll let you readd it.
+
+This proposal aims to accomplish the goals as requested:
+* What brigmedics should do
+* What brigmedics should not do
+* How does the game make sure those things don't happen
+* How brigmedics should influence the round.
+
+Some thoughts on how to go about fixing the issues are also provided in the PR's thread, which will will be incorporated into this proposal.
+
+*[Jun 4, 2023](https://github.com/space-wizards/space-station-14/pull/17037):* The removal is partially reverted again, bringing back the starting gear loadout for admin use.
+
+## Fixing the problems
+
+> ### This is the same role except with security access to boot with none of the responsibilities of a medic, and it's primary use seemed to be security with a hardsuit, not an actual medic.
+
+We should start off by changing the mindset of brigmedic to focus more on medic instead of brig; it should be brig**medic**, not **brig**medic. Brigmedic should be removed from the security department, and be considered a member of the medical department instead. The brigmedic would still have access to the brig and security channels, similar to the civillian department's lawyer, but just like the lawyer is not a true part of security (i.e., they should not have lethals or cuffs outside of emergencies).
```

Comment:
One of the problems I have with having the brigmedic be an entirely civilian based role is the potential for this role to become an antagonist.

The brigmedic will be commonly assisting security in times of dire need. If antagonists play into this and turn right when security needs them most, people will be extremely distrustful of the brigmedic with due time.

IMO this role shouldn't be eligible to roll antagonist, if it's tied into security work this much. Brigmedic should spawn in security and be a part of security, participate in meetings, get to know the department, etc.

I do agree that they shouldn't be carrying cuffs or batons at all. They shouldn't be doing security job. Thus, you need to give them equipment that encourages the push into being a true medic rather than a secoff. Belt filled with medical equipment for example, that doesn't except cuffs or a baton, like what the medical department gets.